### PR TITLE
#294 Updated "listing" component after "container-ripe" changes

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-components-vue/issues/294 |
| Dependencies | https://github.com/ripe-tech/ripe-components-vue/pull/297 |
| Decisions | Updated `listing` component to support the new changes in `container-ripe` |
| Animated GIF | **Before this change:**<br>imagemimagem<br><br>**After this change:**<br>imagemimagem|

### CODE REVIEW

File: vue/components/ui/organisms/listing/listing.vue [hunk 1]
```diff
@@ -5,26 +5,28 @@
...
+            v-on:header-button:click="onHeaderButtonClick"
+        >
+            <slot v-bind:name="slot" v-for="slot in Object.keys($slots)" v-bind:slot="slot" />
+            <template
+                v-for="slot in Object.keys($scopedSlots)"
+                v-bind:slot="slot"
+                slot-scope="scope"
+            >
+                <slot v-bind:name="slot" v-bind="scope" />
+            </template>
```
Comment (gcandal) [inline vue/components/ui/organisms/listing/listing.vue:21]: Use the specific slot names

**Comment by joamag** [comment]
The solution seems to complex, I'm not sure about this one. I need to be convinced this makes sense and is the proper solution to the problem. @gcandal need to talk about this

**Comment by joamag** [comment]
@gcandal @NFSS10 why is this a bug and an enhancement at the same time...

**Comment by joamag** [comment]
@gcandal Let's use monday time slot to re-discuss this one too

**Comment by NFSS10** [comment]
> @gcandal @NFSS10 why is this a bug and an enhancement at the same time...

**Enhancement** as it supports the newly added `container-ripe` features.
**Bug** because it fixes some responsiveness problems that can be seen for example `price-rules` buttons when in mobile.

**Comment by joamag** [comment]
> > @gcandal @NFSS10 why is this a bug and an enhancement at the same time...
> 
> **Enhancement** as it supports the newly added `container-ripe` features.
> **Bug** because it fixes some responsiveness problems that can be seen for example `price-rules` buttons when in mobile.

Nice 👍 , @gcandal maybe we should open a bug for the "bug" part just have some quick GIF "explaining" the issue

**Comment by gcandal** [comment]
> > > @gcandal @NFSS10 why is this a bug and an enhancement at the same time...
> > 
> > 
> > **Enhancement** as it supports the newly added `container-ripe` features.
> > **Bug** because it fixes some responsiveness problems that can be seen for example `price-rules` buttons when in mobile.
> 
> Nice 👍 , @gcandal maybe we should open a bug for the "bug" part just have some quick GIF "explaining" the issue

@NFSS10 given that you were the one finding it, please step forward and give yourself the honor 🎖️

**Pull request merged**